tests/Base/Producer/RecordValidatorTest.php

```diff
@@ -61,4 +66,17 @@ public function invalidRecordThrowsExceptionDataProvider(): array
             ],
         ];
     }
+
+    /**
+     * @doesNotPerformAssertions
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.UselessDocComment
```

Comment:
Please don't suppress CS violations

Reply:
I took it from the test above

Reply:
Ohh I get it, it's because we're using an old version of `doctrine/coding-standard`. Let's keep it for now and I created #216 to handle that. Thanks!